db/migrate/20181026182144_add_tasks_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddTasksTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :tasks, :id => :bigserial do |t|
+      t.string :name
+      t.string :status
+      t.string :state
```

Comment:
What is the difference between status and state?

Reply:
Status would be e.g. ["success", "error", "warning"] and state could be ["queued", "running", "finished"]